src/components/Button/Button.js

```diff
@@ -1,10 +1,18 @@
 import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
 
 class Button extends Component {
   static propTypes = {
+    animated: PropTypes.bool,
+    basic: PropTypes.bool,
     children: PropTypes.string,
+    color: PropTypes.string,
+    emphasis: PropTypes.string,
+    fluid: PropTypes.bool,
+    inverted: PropTypes.bool,
+    loading: PropTypes.bool,
     onClick: PropTypes.func,
-    type: React.PropTypes.oneOf(['button', 'submit']),
+    type: React.PropTypes.oneOf(['button', 'submit', 'reset']),
```

Comment:
I don't think anyone uses reset button types very often, but there are only three types of buttons, so why not.